Content.Server/Speech/Components/DamagedSiliconAccentComponent.cs

```diff
@@ -0,0 +1,90 @@
+﻿using Content.Shared.FixedPoint;
+namespace Content.Server.Speech.Components;
+
+[RegisterComponent]
+public sealed partial class DamagedSiliconAccentComponent : Component
+{
```

Comment:
Shared + predicted, just because chat isn't doesn't mean we can't do it here in anticipation.

Reply:
Wouldn't this require synchronizing the state of the random generator? If so, I'm guessing I'll need to seed the random generator on the tick value or something like in #34863.

Going to work on this, let me know if I'm barking up the wrong tree...

Reply:
Yeah I'm kind of unsure how to approach this, because battery charge levels and total damage need to be used to calculate the corruption on the client side, and I don't think those values are accessible client side, right?

Reply:
Resolved per discord conversation hopefully